### DISCUSSION TITLE
is cocotb_tools a new package?

### DISCUSSION THREAD

**Discussion by psumesh** [cocotb/cocotb#3970] [Q&A] [answered]
Hi,

is cocotb_tools.runner a new package ?

**Comment by ktbarrett** [comment, marked as answer]
Yes, it's a new Python package. It's located in the cocotb pip package. pip packages can contain multiple Python packages.

**Reply by Yuchen-He1** [reply]
Hi !
I try to run the test_adder example on cocotb 1.9
I have a question about why cocotb_tools can not find here since it's already located in the cocotb pipi package?

![Screenshot 2025-01-31 201603](https://github.com/user-attachments/assets/db214527-6033-4171-bd65-3852a6c239e7)

**Reply by cmarqu** [reply]
@Yuchen-He1 You need to get the example code from the tag that matches your cocotb installation, in your case from https://github.com/cocotb/cocotb/tree/stable/1.9/examples/adder/tests
